### ISSUE TITLE
Is this a bug? Yes this appears to be a bad bug.

### ISSUE THREAD

**Issue by jeffwright13** [jeffwright13/pytest-tui#67]
plugin.py:

`if not (hasattr(config.option, "tui") and config.option.tui)`

Execution args:

```            "args": [
                "-v",
                "--tui",
                "--skip-prepare",
                "--core-test",
                "-k",
                "test_device_energy_counters",]

**Comment by jeffwright13** [comment]
```$  cd /Users/jwr003/repos/gqa/aims_la_tests ; /usr/bin/env /Users/jwr003/repos/gqa/venv/bin/python /Users/jwr003/.vscode/extensions/ms-python.python-2022
.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher 59120 -- /Users/jwr003/repos/gqa/aims_la_tests/__main__.py -v --tui --skip-prepare --
core-test -k test_device_energy_counters 
Traceback (most recent call last):
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/main.py", line 267, in wrap_session
    config.hook.pytest_sessionstart(session=session)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 60, in _multicall
    return outcome.get_result()
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/Users/jwr003/repos/gqa/aims_la_tests/conftest.py", line 322, in pytest_sessionstart
    LOGGER.info("Conftest Session start")
  File "/Users/jwr003/.pyenv/versions/3.9.9/lib/python3.9/logging/__init__.py", line 1446, in info
    self._log(INFO, msg, args, **kwargs)
  File "/Users/jwr003/.pyenv/versions/3.9.9/lib/python3.9/logging/__init__.py", line 1589, in _log
    self.handle(record)
  File "/Users/jwr003/.pyenv/versions/3.9.9/lib/python3.9/logging/__init__.py", line 1599, in handle
    self.callHandlers(record)
  File "/Users/jwr003/.pyenv/versions/3.9.9/lib/python3.9/logging/__init__.py", line 1661, in callHandlers
    hdlr.handle(record)
  File "/Users/jwr003/.pyenv/versions/3.9.9/lib/python3.9/logging/__init__.py", line 952, in handle
    self.emit(record)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/logging.py", line 794, in emit
    self.stream.write("\n")
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/terminal.py", line 421, in write
    self._tw.write(content, flush=flush, **markup)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pytest_tui/plugin.py", line 181, in tee_write
    if config._tui_sessionstart:
AttributeError: 'Config' object has no attribute '_tui_sessionstart'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/main.py", line 289, in wrap_session
    config.notify_exception(excinfo, config.option)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/config/__init__.py", line 1037, in notify_exception
    res = self.hook.pytest_internalerror(excrepr=excrepr, excinfo=excinfo)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 60, in _multicall
    return outcome.get_result()
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/terminal.py", line 474, in pytest_internalerror
    self.write_line("INTERNALERROR> " + line)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/terminal.py", line 430, in write_line
    self._tw.line(line, **markup)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/_io/terminalwriter.py", line 170, in line
    self.write(s, **markup)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pytest_tui/plugin.py", line 181, in tee_write
    if config._tui_sessionstart:
AttributeError: 'Config' object has no attribute '_tui_sessionstart'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jwr003/.pyenv/versions/3.9.9/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/jwr003/.pyenv/versions/3.9.9/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/jwr003/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 39, in <module>
    cli.main()
  File "/Users/jwr003/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 430, in main
    run()
  File "/Users/jwr003/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/Users/jwr003/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/Users/jwr003/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/Users/jwr003/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "/Users/jwr003/repos/gqa/aims_la_tests/__main__.py", line 15, in <module>
    main()
  File "/Users/jwr003/repos/gqa/aims_la_tests/__main__.py", line 11, in main
    pytest.main()
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/config/__init__.py", line 162, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 60, in _multicall
    return outcome.get_result()
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/main.py", line 316, in pytest_cmdline_main
    return wrap_session(config, _main)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/main.py", line 311, in wrap_session
    config._ensure_unconfigure()
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/_pytest/config/__init__.py", line 987, in _ensure_unconfigure
    self.hook.pytest_unconfigure(config=self)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 60, in _multicall
    return outcome.get_result()
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/Users/jwr003/repos/gqa/venv/lib/python3.9/site-packages/pytest_tui/plugin.py", line 259, in pytest_unconfigure
    _tui_terminal_out.seek(0)
ValueError: seek of closed file

**Comment by jeffwright13** [comment]
Should be fixed with 1.4.2.
